### PR TITLE
added schema for projects

### PR DESCRIPTION
**Pull request by Rupeshiya**
Fix for #70 
@devesh-verma @vaibhavdaren Please review.

### CODE REVIEW

File: app/models/Project.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      }
+    },
+    long: {
+      type: String,
+      trim: true
+    }
+  },
+  image: {
+    type: Buffer,
+    contentType: String
```
Comment (devesh-verma) [inline app/models/Project.js]: add one more field to this - `imgUrl` -> this will contain aws s3 path of the img

Comment (Rupeshiya) [reply app/models/Project.js]: done! Let me know if any more changes required!

File: app/models/Project.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+          throw new Error('Short description should be min 10 characters long!')
+        }
+      }
+    },
+    long: {
+      type: String,
+      trim: true
+    }
+  },
+  image: {
```
Comment (devesh-verma) [inline app/models/Project.js]: in one project there can be multiple images, how are we handling that?

Comment (devesh-verma) [reply app/models/Project.js]: its okay for now we can take this up at later point in time

Comment (Rupeshiya) [reply app/models/Project.js]: oh, then we should create an array of imageIds, I think aws create unique ids for each image?
and during rendering we can just fetch the image according to the ids.
What do you think?

**Pull request merged**